homeassistant/components/rfxtrx/__init__.py

```diff
@@ -52,30 +57,28 @@
         ("Temperature", TEMP_CELSIUS),
         ("Temperature2", TEMP_CELSIUS),
         ("Humidity", PERCENTAGE),
-        ("Barometer", ""),
+        ("Barometer", PRESSURE_HPA),
         ("Wind direction", ""),
-        ("Rain rate", ""),
+        ("Rain rate", "mm/h"),
         ("Energy usage", POWER_WATT),
-        ("Total usage", POWER_WATT),
+        ("Total usage", ENERGY_KILO_WATT_HOUR),
         ("Sound", ""),
```

Comment:
No unit of measurement should be represented by `None`.

Reply:
I assume you are commenting on the empty string for sound? 

There is a difference between none and empty thou.
Without unit, home assistant won't show graphs for numeric values. We know they are numeric but we don't know their unit.

Sound is just a number between 0 and 9. Sound specifically can be None thou since it's not a continuous value.

Reply:
They need to be in the dict, since it's a filter for values.

Reply:
We shouldn't return empty string just to get a graph in the frontend. If you want to show a graph without unit, I think we should make that suggestion to the frontend team and see how we can accomplish that.

Reply:
I swapped them all to None. There are two cases where it would be good to have a unit of just numeric type. Ie an untyped counter. Do we have such a unit?

Reply:
There's no standard. Maybe `count`?